app/controllers/concerto_plugins_controller.rb

```diff
@@ -106,14 +106,22 @@ def destroy
   end
 
   def update_gem
+    results = {}
     plugin = ConcertoPlugin.find(params[:id])
-    system('bundle update', plugin.gem_name)
-    rake_precompile()
-    restarted = restart_webserver()
-    if restarted
-      flash[:notice] = t(:plugin_updated)
+    results[:bundle_output] = `bundle update #{plugin.gem_name}`
```

Comment:
@augustf should any of the parameters used in the bundle install from the boot.rb line 57 be included here?